mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Postgrex.Mixfile do
   def project do
     [app: :postgrex,
      version: "0.9.2-dev",
-     elixir: "~> 1.0",
+     elixir: "~> 1.1",
```

Comment:
Please let's not reimplement GenServer.whereis. What if we leave it as 1.0 and say that particular feature requires 1.1?